### PR TITLE
Added documentation on ReadTheDocs

### PR DESCRIPTION
**Pull request by jcohenadad**
This PR adds documentation to the project.

Latest build (pointing to this branch) is [here](https://ivado-medical-imaging.readthedocs.io/en/jca-doc/).

- Documentation is built by [ReadTheDocs](https://readthedocs.org/) for each new event;
- Documentation uses the MarkDown format, powered by Sphinx-compatible [Recommonmark](https://github.com/readthedocs/recommonmark);
- The procedure to build the documentation is described in the [wiki](https://github.com/neuropoly/ivado-medical-imaging/wiki/documentation)
- Added API documentation
- Fixed ivado logo display on readme

TODO in other PRs:
- Add docstrings to all methods: https://github.com/neuropoly/ivado-medical-imaging/issues/250
- Add documentation build in CI: https://github.com/neuropoly/ivado-medical-imaging/issues/249
- Redirect ivadomed.org to RTD latest version and add link to README at the top of the github's project page: https://github.com/neuropoly/ivado-medical-imaging/issues/248
- Find a way to fetch all classes/functions per file automatically: https://github.com/neuropoly/ivado-medical-imaging/issues/247

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,120 +3,15 @@
...
-  <img src="https://github.com/neuropoly/ivado-medical-imaging/raw/master/images/mila_logo.png" height="120" /> 
-  <img src="https://github.com/neuropoly/ivado-medical-imaging/raw/master/images/ivado_logo.png" height="120" />
+  <img src="images/neuropoly_logo.png" height="80" />
+  <img src="images/mila_logo.png" height="80" />
+  <img src="images/ivado_logo.png" height="80" />
 </p>
 
 # IVADO Medical Imaging
 Comprehensive and open-source repository of deep learning methods for medical data segmentation.
 Collaboration between MILA and NeuroPoly for the IVADO project on medical imaging.
```
Comment (andreanne-lemay) [inline README.md:14]: Do we want to include a link to RTD here? Even if it's not final, people visiting our repo could have access to the doc.

File: docs/source/data.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Data
+
+Without data, nothing can be done. To get you started, we recommend you download
+the [Spinal Cord MRI Public Database](https://openneuro.org/datasets/ds001919).
```
Comment (andreanne-lemay) [inline docs/source/data.md:4]: We could also add a link to toy dataset (only a few images vs. 8 Go dataset). We could then use this small dataset in our tutorial code.

File: docs/source/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+New model can be generated using the command-line tool from the terminal:
+
+```
+ivadomed config/config.json
+```
+
+where `config.json` is a configuration file, which parameters are described [here](https://github.com/neuropoly/ivado-medical-imaging/wiki/configuration-file).
+
+To fully benefits from all the features of `ivadomed`, please see our [tutorials](tutorials.md).
```
Comment (andreanne-lemay) [inline docs/source/getting_started.md:11]: To full benefits -> To fully benefit

File: docs/source/overview.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+We acknowledge the existence of similar projects. The specificity of `ivadomed`
+is highlighted in the table below:
+
+``` warning:: TODO: Add comparative table
+```
+
+## Features
+
+### Physic-informed network
+CNNs can be modulated, at each layer, using the [Feature-wise Linear Modulation (FiLM)](https://arxiv.org/pdf/1709.07871.pdf)) technique. FiLM permits to add priors during training/inference
```
Comment (andreanne-lemay) [inline docs/source/overview.md:19]: Double parenthesis (FiLM))

File: docs/source/getting_started.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Getting started
+
+New model can be generated using the command-line tool from the terminal:
+
+```
+ivadomed config/config.json
```
Comment (charleygros) [inline docs/source/getting_started.md:6]:
```suggestion
ivadomed ivadomed/config/config.json
```

File: .gitignore [hunk 6]
```diff
@@ -7,4 +7,4 @@ ivadomed/FiLM/__pycache__/
 dev/.bin/
 dev/__pycache__/
 testing/__pycache__/
-
+docs/build
```
[the remark this answers is not in the document]
Comment (jcohenadad) [reply .gitignore:10]: i also go the same warnings. Issue opened: https://github.com/neuropoly/ivado-medical-imaging/issues/252

File: README.md (hunk 1 above)
Comment (jcohenadad) [reply README.md:14]: step-by-step: https://github.com/neuropoly/ivado-medical-imaging/issues/248

File: docs/source/data.md (hunk 2 above)
Comment (jcohenadad) [reply docs/source/data.md:4]: Good point-- issue opened: https://github.com/neuropoly/ivado-medical-imaging/issues/253

File: docs/source/getting_started.md (hunk 3 above)
Comment (jcohenadad) [reply docs/source/getting_started.md]: fixed

File: docs/source/overview.md (hunk 4 above)
Comment (jcohenadad) [reply docs/source/overview.md]: fixed

**Comment by jcohenadad** [comment]
thanks for the review @andreanne-lemay @charleygros !

**Pull request merged**